src/cli.rs

```diff
@@ -188,6 +193,19 @@ fn parse_method_url(s: &str) -> Result<(Option<Method>, String)> {
     }
 }
 
+fn validate_session_name(s: &str) -> Result<String> {
```

Comment:
If we aren't tying session names (identifiers? unclear due to seemingly-mixed terminology) to paths, do we want to validate that the name is a valid path?

Reply:
Yes, in theory if we choose to use hashes we shouldn't need to validate the name of the session (which is the filename no more) for being a valid path.  Plus, it clears the context of both name and identifier. Identifier is the user-defined name of the session, and the filename is its calculated hash.